lib/wallaroo/application.pony

```diff
@@ -82,15 +85,19 @@ class Pipeline[Out: Any val] is BasicPipeline
     let sc_wrapper = SourceConfigWrapper(n, source_config)
     let source_id' = _stages.add_node(sc_wrapper)
     _dag_sink_ids.push(source_id')
+    _worker_source_configs.update(sc_wrapper.name(),
```

Comment:
This should be:
`_worker_source_configs(sc_wrapper.name()) = source_config.worker_source_config()`